assets/js/base/components/chip/stories/chip.stories.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { Story, Meta } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import Chip, { ChipProps } from '../chip';
+const availableElements = [ 'li', 'div', 'span' ];
+const availableRadius = [ 'none', 'small', 'medium', 'large' ];
```

Comment:
As a descendant of the ancient Romans, I feel compelled to say that the plural of `radius` is `radii` 😂 

Just kidding of course, I just couldn't stop myself. It's like when Italians see people cooking pasta, they always *have to* say something.

Reply:
Haha I too am a descendent of the Romans, and I bow my head in shame at the atrocity committed here. I will correct this injustice at once.